test/e2e/specs/editor/plugins/iframed-masonry-block.spec.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'iframed masonry block', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-iframed-masonry-block'
+		);
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-iframed-masonry-block'
+		);
+	} );
+
+	test( 'should load script and dependencies in iframe', async ( {
+		editor,
+	} ) => {
+		await editor.insertBlock( { name: 'test/iframed-masonry-block' } );
+
+		const masonry = editor.canvas.getByRole( 'document', {
+			name: 'Block: Iframed Masonry Block',
+		} );
+		await expect( masonry ).toBeVisible();
+
+		const masonryHeight = await masonry.evaluate( ( node ) =>
```

Comment:
Curious to know whether we can use `boundingBox` here?

Reply:
Good idea! Updated in [9a66431](https://github.com/WordPress/gutenberg/pull/55016/commits/9a66431a2f755331fcef34bd1b9f0fe8428f1dcd).